cappy.py

```diff
@@ -123,11 +137,12 @@ def handle_inbound_request(self, message):
             request_id = message['id']
             args = message['args']
         except KeyError as e:
+            print(e)
```

Comment:
Makes sense. I'll remove the `e` for now but leave the TODO comment so that later on we can re-implement it.